packages/remix-dev/config/defaults/entry.server.cloudflare.tsx

```diff
@@ -0,0 +1,32 @@
+import type { EntryContext } from "@remix-run/cloudflare";
+import { RemixServer } from "@remix-run/react";
+import isbot from "isbot";
```

Comment:
we should also check and install `isbot` if it's not already, then log that they should commit the change